features/import.feature

```diff
@@ -87,3 +87,61 @@ Feature: Import content.
       """
       100
       """
+
+  Scenario: Export and import page and referencing menu item in separate files
+    Given a WP install
+    And I run `mkdir export`
+
+    When I run `wp menu create "My Menu"`
+    And I run `wp menu item add-post my-menu 2`
+    And I run `wp menu item list my-menu --format=count`
+    Then STDOUT should be:
+      """
+      1
+      """
+
+    When I run `wp export --dir=export --post_type=page --filename_format=0.page.xml`
+    When I run `wp export --dir=export --post_type=nav_menu_item --filename_format=1.menu.xml`
+    Then STDOUT should not be empty
+
+    When I run `wp site empty --yes`
+    Then STDOUT should not be empty
+
+    When I run `wp post list --post_type=page --format=count`
+    Then STDOUT should be:
+      """
+      0
+      """
+
+    When I run `wp post list --post_type=nav_menu_item --format=count`
+    Then STDOUT should be:
+      """
+      0
+      """
+
+    When I run `find export -type f | wc -l`
+    Then STDOUT should be:
+      """
+             2
+      """
+
+    When I run `wp plugin install wordpress-importer --activate`
+    Then STDERR should not contain:
+      """
+      Warning:
+      """
+
+    When I run `wp import export --authors=skip --skip=image_resize`
+    Then STDOUT should not be empty
+
+    When I run `wp post list --post_type=page --format=count`
+    Then STDOUT should be:
+      """
+      1
+      """
+
+    When I run `wp post list --post_type=nav_menu_item --format=count`
```

Comment:
Can we explicitly check that this is the right menu item, and that it's properly connected to the post?